CPS scope selector changes (#7815)
elastic/docs-content: deploy-manage/cross-project-search-config/cps-config-access-and-scope.md

2. In the general space settings, find the **{{cps-cap}}** panel and set the default scope for the space:
   - **All projects:** (default) Searches run across the origin project and all linked projects.
   - **This project:** Searches run only against the origin project's data.
   - **Specific projects:** Use the [scope selector](/explore-analyze/cross-project-search/cross-project-search-manage-scope.md#cps-in-kibana) to include or exclude individual projects or filter by [project tags](/explore-analyze/cross-project-search/cross-project-search-tags.md).

3. Click **Apply changes** to save the scope setting.

A **Specific projects** default is saved as a [project routing expression](/explore-analyze/cross-project-search/cross-project-search-project-routing.md), not a fixed list, so it also applies to projects you link later:

- Projects that match your tag filters are included automatically.
- Excluding a project individually affects only that project.

> To keep future projects out of the default, use a tag filter rather than excluding projects one by one.

Reviewer: can we add something around add/removing/editing project tags may affect project defaults.

> To keep future projects out of the default, use a tag filter rather than excluding projects one by one.

Author: ```suggestion
To keep future projects out of the default, use a tag filter rather than excluding projects one by one.

:::{note}
When you use a tag filter, changing a project's [custom tags](/deploy-manage/deploy/elastic-cloud/project-settings.md#project-tags) can include or exclude that project from the default, without editing the space setting.
:::
```

::::{note}

> The default {{cps}} scope is a space setting, not an access control. Users can still set the scope at the query level. You can also [manage user access](#manage-user-and-api-key-access).

Reviewer: > Users can still set the scope at the query level.
also with the project picker from the UI

   - **This project:** Searches run only against the origin project's data.
   - **Specific projects:** Use the [scope selector](/explore-analyze/cross-project-search/cross-project-search-manage-scope.md#cps-in-kibana) to include or exclude individual projects or filter by [project tags](/explore-analyze/cross-project-search/cross-project-search-tags.md).

3. Click **Apply changes** to save the scope setting.

A **Specific projects** default is saved as a [project routing expression](/explore-analyze/cross-project-search/cross-project-search-project-routing.md), not a fixed list, so it also applies to projects you link later:

- Projects that match your tag filters are included automatically.
- Excluding a project individually affects only that project.

To keep future projects out of the default, use a tag filter rather than excluding projects one by one.

:::{note}
When you use a tag filter, changing a project's [custom tags](/deploy-manage/deploy/elastic-cloud/project-settings.md#project-tags) can include or exclude that project from the default, without editing the space setting.
:::

::::{note}

> The default {{cps}} scope is a space setting, not an access control. Users can still set the scope at the query level. You can also [manage user access](#manage-user-and-api-key-access).

Author: ```suggestion
The default {{cps}} scope is a space setting, not an access control. Depending on the app, users can still change scope with the [{{cps-init}} scope selector](/explore-analyze/cross-project-search/cross-project-search-manage-scope.md#cps-in-kibana) or at the query level. You can also [manage user access](#manage-user-and-api-key-access).
```

   - **All projects:** (default) Searches run across the origin project and all linked projects.
   - **This project:** Searches run only against the origin project's data.
   - **Specific projects:** Use the [scope selector](/explore-analyze/cross-project-search/cross-project-search-manage-scope.md#cps-in-kibana) to include or exclude individual projects or filter by [project tags](/explore-analyze/cross-project-search/cross-project-search-tags.md).

3. Click **Apply changes** to save the scope setting.

A **Specific projects** default is saved as a [project routing expression](/explore-analyze/cross-project-search/cross-project-search-project-routing.md), not a fixed list, so it also applies to projects you link later:

- Projects that match your tag filters are included automatically.
- Excluding a project individually affects only that project.

To keep future projects out of the default, use a tag filter rather than excluding projects one by one.

:::{note}
When you use a tag filter, changing a project's [custom tags](/deploy-manage/deploy/elastic-cloud/project-settings.md#project-tags) can include or exclude that project from the default, without editing the space setting.

> :::

Author: ```suggestion
To keep future projects out of the default, use a tag filter rather than excluding projects one by one. When you use a tag filter, changing a project's [custom tags](/deploy-manage/deploy/elastic-cloud/project-settings.md#project-tags) can include or exclude that project from the default, without editing the space setting.
```
